modules/Rock.cmake

```diff
@@ -49,6 +49,31 @@ function(rock_add_compiler_flag_if_it_exists FLAG)
     endif()
 endfunction()
 
+macro(ADD_TARGET_PROPERTIES _target _name _properties)
```

Comment:
prefix all macros defined here by 'rock_', this is to make sure we don't collide with other CMake macro sets.

And stick to the lowercase naming style.